profile/README.md

```diff
@@ -1,6 +1,6 @@
 ### Hi there, this is n8n 👋
 
-n8n is a low-code automation tool, and with over 220 pre-built integrations and a general connector for anything with an API, n8n enables you to connect anything to everything. With n8n you can move beyond simple one step integrations to build multi-step workflows that integrate your tools exactly the way you want. Thanks to its <a href="https://faircode.io/">fair-code</a> distribution model, n8n will always have visible source code, be available to self-host, is free to use and allows you to add your own custom functions, logic, and apps.
+n8n is a low-code automation tool. With over 220 pre-built integrations and a general connector for anything with an API, n8n enables you to connect anything to everything. With n8n you can move beyond simple one step integrations to build multi-step workflows that integrate your tools exactly the way you want, including adding AI functionality. Thanks to its <a href="https://faircode.io/">fair-code</a> distribution model, n8n will always have visible source code, be available to self-host, have a free community addition, and allow you to add your own custom functions, logic, and apps.
```

Comment:
> have a free community addition

Should that be "edition"?

Reply:
Yes it should. I've added a comment with the change, but I can't actually accept it into the PR.